### PR TITLE
APT-3572: Setting radar modes via DDS is now supported

### PR DESCRIPTION
**Pull request by iurii-provizio**
Used while in this branch and works very well now

### CODE REVIEW

File: provizio_dds_idls_fastdds/provizio/msg/radar_info.cxx [hunk 1]
```diff
@@ -35,25 +35,26 @@ using namespace eprosima::fastcdr::exception;
...
 
     // string m_serial_number
     m_serial_number ="";
-    // sequence<provizio::msg::radar_mode> m_supported_modes
+    // sequence<provizio::msg::radar_range> m_supported_ranges
 
-    // provizio::msg::radar_mode m_current_mode
-    m_current_mode = provizio::msg::short_range;
+    // provizio::msg::radar_range m_current_range
+    m_current_range = provizio::msg::short_range;
```
Comment (psilar) [inline provizio_dds_idls_fastdds/provizio/msg/radar_info.cxx:57]: Is a default range range being specified here?  If so, I think the default should be medium range.

File: provizio_dds_idls_fastdds/provizio/msg/radar_range.cxx [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+
+
+provizio::msg::set_radar_range::set_radar_range()
+{
+    // std_msgs::msg::Header m_header
+
+    // string m_serial_number
+    m_serial_number ="";
+    // provizio::msg::radar_range m_target_range
+    m_target_range = provizio::msg::short_range;
```
Comment (psilar) [inline provizio_dds_idls_fastdds/provizio/msg/radar_range.cxx:54]: Again, default to medium range?

**Review by psilar — APPROVED** [review]
Minor comment relating to the default (?) range_mode but not a show-stopper.

File: provizio_dds_idls_fastdds/provizio/msg/radar_range.cxx (hunk 2 above)
Comment (iurii-provizio) [reply provizio_dds_idls_fastdds/provizio/msg/radar_range.cxx:54]: @psilar it's just 0, and this part is not in my control - it's auto-generated. Everywhere I deal with radar ranges, I do set them to something meaningful depending on the context. Most of the the time it's unkown (but here unkown is not an option, and for a good reason - the way UDP protocol deals with ranges there are ways for it to be in "not-yet-known" state). In DDS protocol, either you haven't heard anything about ranges yet, or the range is known and specific.

**Pull request merged**